scripts/gha/build_desktop.py

```diff
@@ -37,7 +37,23 @@
 import argparse
 import os
 import utils
+import shutil
```

Comment:
This file will be a fun one to merge with my latest changes. I feel we should definitely get your PR in first and then layer mine over it. I will take care of handling conflicts in this file.

Reply:
SG, thanks!